Formula/iso-codes.rb

```diff
@@ -14,9 +14,18 @@ class IsoCodes < Formula
   end
 
   depends_on "gettext" => :build
+  depends_on :python3
+  depends_on "pkg-config" => :run
 
   def install
     system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "check"
     system "make", "install"
   end
+
+  test do
+    output = shell_output("#{Formula["pkg-config"].opt_bin}/pkg-config --variable=domains iso-codes")
+    assert_match /iso_639-2 iso_639-3 iso_639-5 iso_3166-1/, output
```

Comment:
Please use a string (as already suggested by @DomT4) instead of a regular expression when a simple substring match is sufficient.